<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QArkPlotPenSettingsWidget</class>
 <widget class="QWidget" name="QArkPlotPenSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Couleur et style</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QArkClickableFilledRect" name="colorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="redLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="redSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>Epaisseur</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="widthSpinBox"/>
      </item>
      <item row="1" column="0" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>45</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="greenLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>G</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="greenSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="styleLabel">
        <property name="text">
         <string>Style</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="styleComboBox"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="blueLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="blueSpinBox">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="applyPushButton">
        <property name="text">
         <string>Appliquer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QArkClickableFilledRect</class>
   <extends>QLabel</extends>
   <header>pyQArk.Widgets.QArkClickableFilledRect.QArkClickableFilledRect.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
